### ISSUE TITLE
Incorrect SHA error using brew install

### ISSUE THREAD

**Issue by SteveDMurphy** [cantino/mcfly#312]
https://github.com/cantino/mcfly/blob/03a83e15dc7692a39efd9cdb5f7bbd81afababb2/pkg/brew/mcfly.rb#L15-L16

On an M1 Mac and the install steps are failing with the below:
```
Error: mcfly: SHA256 mismatch
Expected: f7c5e81714fb76d81ef5741d1e43e87ea79da7eb5c30c84ec08e451ca7c9c5d0
Actual: 6649f1356215c8193853d0e4fd00f1686c0d74380a2754fa1ffb60faf34bebed
```

**Comment by cexoso** [comment]
+1

**Comment by cantino** [comment]
Sorry everyone, I believe I've fixed it now. Can you confirm?

**Comment by SteveDMurphy** [comment]
Closed by [745d7f4](https://github.com/cantino/mcfly/commit/745d7f4bc74976df3080b850bb37d130ac0f6aa7)
